src/modules/hardwareWallet/store/selectors/hwSelectors.js

```diff
@@ -0,0 +1,3 @@
+export const selectHardwareDevices = (state) => state.hardwareWallet.devices;
+export const selectActiveHardwareDeviceId = (state) => state.hardwareWallet.currentDevice.deviceId;
+export const selectActiveHardwareDevice = (state) => state.hardwareWallet.currentDevice;
```

Comment:
@oskarleonard There's already a single file of selectors in `src/redux/selectors.js`. I think these can be added there

Reply:
Fixed by moving the global hw selectors to the hw module